Pokedex_iosTests/TestResources.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  TestResources.swift
+//  Pokedex_iosTests
+//
+//  Created by Alvaro Choque on 4/7/22.
+//
+
+import Foundation
+@testable import Pokedex_ios
+
+class SampleTest {
```

Comment:
This classname shouldn't be SampleTest as it is being used, it should be Something more descriptive.

Reply:
seems good to me